xpk.py

```diff
@@ -4243,7 +4243,7 @@ def cluster_delete(args) -> int:
   if run_gke_cluster_delete_command_code != 0:
     xpk_exit(run_gke_cluster_delete_command_code)
   xpk_print(f'GKE commands done! Cluster {args.cluster} deleted.\n')
-  return 0
+  return xpk_exit(0)
```

Comment:
xpk_exit doesn't return any value.

Probably what you want is without the return statement:
`xpk_exit(0)`

---

Most of the subcommand functions like cluster_delete are actually misleading to say they return int. They actually never return. Some longer term clean up exists there.

Reply:
Aah, makes sense. Thank you